docker-compose.yml

```diff
@@ -453,6 +561,45 @@ services:
         <<: *depends_on-healthy
       web:
         <<: *depends_on-healthy
+    profiles:
+      - feature-complete
+  nginx-errors-only:
+    <<: *restart_policy
+    ports:
+      - "$SENTRY_BIND:80/tcp"
+    image: "nginx:1.25.4-alpine"
+    volumes:
+      - type: bind
+        read_only: true
+        source: ./nginx
+        target: /etc/nginx
+      - sentry-nginx-cache:/var/cache/nginx
+    depends_on:
+      - web-errors-only
+      - relay-errors-only
+    profiles:
+      - errors-only
+  relay-errors-only:
```

Comment:
Shouldn't relay behave the same for errors-only and feature-complete ones?

Reply:
Yes, it pretty much does. However, I wasn't able to get to a clean solution here with the docker compose architecture and wanted to avoid merging docker compose files. The difference is the `depends_on` clause for docker compose. For relay-errors-only, we it depends on `web-errors-only`, which has a different set of services it depends on to be considered healthy.

Reply:
Ah got it